### PR TITLE
PRO-1855 documentation for messaging system

### PR DESCRIPTION
**Pull request by tkfu**

### CODE REVIEW

File: docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+This message is sent when the status of a package update on a particular device changes.
+
+Fields:
+
+[%autoscale,cols="1,4",width="60%"]
+|===
+| namespace   | A string representing the namespace subscribed to
+| device      | The uuid of the device in question
+| packageUuid | The UUID of the package in question
```
Comment (jerrytrieu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: 'UUID' vs 'uuid' previously

File: docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc [hunk 2]
```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: "Messaging in rvi_sota_server"
+category: dev
+order: 6
+date: 2016-12-30 13:59:16
+---
+
+Communication between rvi_sota_server's microservices happens over a message bus. Any of the microservices can publish messages to the bus, and other microservices can subscribe to the messages they are interested in. For example, Resolver keeps a database of packages, and thus needs to know when a new package is created. When a package is uploaded via Core's PUT /packages endpoint, Core publishes a PackageCreated message, so that Resolver can remain consistent.
```
Comment (jerrytrieu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: This example will be out-of-date soon when we remove packages from resolver.

File: docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+Fields:
+
+[%autoscale,cols="1,4",width="60%"]
+|===
+| namespace| A string representing the namespace subscribed to
+| uuid     | The uuid of the device in question
+| at       | A UTC timestamp of the time the device first connected
+|===
+
+=== DeviceDeleted
```
Comment (jerrytrieu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: Let's not document this yet.

Comment (tkfu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: DeviceDeleted, or DeviceActivated?

File: docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc (hunk 2 above)
Comment (jerrytrieu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: Ok, can update later then.

File: docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc (hunk 3 above)
Comment (jerrytrieu) [inline docs/_posts/2016-12-30-messaging-in-rvisotaserver.adoc]: DeviceDeleted

**Pull request merged**